tests/unit/lms/services/group_info_test.py

```diff
@@ -119,7 +119,7 @@ def params(self):
         return {
             column: f"TEST_{column.upper()}"
             for column in GroupInfo.columns()
-            if column not in ("consumer_key", "_info")
+            if column not in ("consumer_key", "_info", "application_instance_id")
```

Comment:
This automatic param generation based on the name only it's not great. 

Noted to revisit the service and test for a refactor when this settles.

Reply:
Eh. :shrug: Maybe this should just be hard-coded?